src/mapping.ts

```diff
@@ -0,0 +1,58 @@
+import { BigInt } from "@graphprotocol/graph-ts"
+import {
+  Registrar,
+  Approval,
+  ApprovalForAll,
+  ControllerAdded,
+  ControllerRemoved,
+  DomainCreated,
+  OwnershipTransferred,
+  Paused,
+  Transfer,
+  Unpaused
+} from "../generated/Registrar/Registrar"
+import { Domain } from "../generated/schema"
+
+//  event DomainCreated(
+//    uint256 indexed id,
+//    string name,
+//    uint256 indexed nameHash,
+//    uint256 indexed parent
+//  );
+//need a feild for owner address
+export function handleDomainCreated(event: DomainCreated): void {
+  let account = new Account(event.params.owner.toHex())
+  account.save()
+
+  //store domain by the hash of its unique token ID
+  let domain = Domain.load(event.params.id.toHex())
+  let domainParent = Domain.load(event.params.parent.toHex())
+
+  if (domain == null) {
+    domain = new Domain(event.params.id.toHex())
+  }
+    domain.owner = event.params.owner
+    domain.creator = event.params.ownerOf
```

Comment:
Just as a heads up, this is going to be changed a bit.
The event has been updated to also publish the creator address.
For now this is good though

Reply:
ok, sweet. Yea I was going to suggest adding that. I think I even left a comment formyself near this line that I was going to bring up to you when you got back.